app-ios/Sources/SessionFeature/SessionTagsView.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
+import appioscombined
+import Theme
+
+extension TimetableItem {
+    
+    var tagsView: some View {
+        let tags = self.tags
+
+        return GeometryReader { geometry in
+            var width = CGFloat.zero
+            var height = CGFloat.zero
+
+            ZStack(alignment: .topLeading) {
+                ForEach(tags, id: \.self) { tag in
+                    tag.view
+                        .padding([.trailing, .bottom], 8)
+                        .alignmentGuide(.leading, computeValue: { dimension in
+                            if abs(width - dimension.width) > geometry.size.width {
+                                width = 0
+                                height -= dimension.height
+                            }
+
+                            let result = width
+                            if tag == tags.last! {
+                                width = 0
+                            } else {
+                                width -= dimension.width
+                            }
+                            return result
+                        })
+                        .alignmentGuide(.top, computeValue: { _ in
+                            let result = height
+                            if tag == tags.last! {
+                                height = 0 // last item
+                            }
+                            return result
+                        })
+                }
+            }
+        }
+    }
+
```

Comment:
Implemented a custom view for overflowing tags. 
But it seems to have dimensions incorrectly set. When I remove it's bottom padding, the view below gets wrongly placed as follows:

<img src="https://user-images.githubusercontent.com/5493586/190045697-02fd4821-0cd8-4eae-a953-88d91feacab7.png" width="250" />

Any hint would be of great help!

Reply:
Thanks for the hint! 👍